packages/core/src/util/calcStabilityWindow.ts

```diff
@@ -0,0 +1,9 @@
+// 3k/f (where k is the security parameter in genesis, and f is the active slot co-efficient parameter
+// in genesis that determines the probability for amount of blocks created in an epoch.)
+
+import { Cardano } from '..';
+
+export const calculateStabilityWindowSlotsCount = ({
+  securityParameter,
+  activeSlotsCoefficient
+}: Cardano.CompactGenesis): number => (3 * securityParameter) / activeSlotsCoefficient;
```

Comment:
The less constrained the utilities are the better
```suggestion
}: Pick<Cardano.CompactGenesis, 'securityParameter' | 'activeSlotsCoefficient'>): number => (3 * securityParameter) / activeSlotsCoefficient;
```

Reply:
c66f5797594c56d155aeb62836e4228622916a55